src/componentsNext/Pagination/__tests__/describePagination.test.ts

```diff
@@ -0,0 +1,137 @@
+import { describe, expect, test } from "vitest";
+
+import describePagination from "../describePagination";
+
+describe("describePagination", () => {
+  describe("it describes a simple 10 page pagination", () => {
+    test("page 1 / 10", () => {
+      const pagination = describePagination({ pages: 10, currentPage: 1 });
+      expect(pagination).toStrictEqual([1, 2, "...", 9, 10]);
+    });
+
+    test("page 2 / 10", () => {
+      const pagination = describePagination({ pages: 10, currentPage: 2 });
+      expect(pagination).toStrictEqual([1, 2, 3, "...", 9, 10]);
+    });
+
+    test("page 5 / 10", () => {
+      const pagination = describePagination({ pages: 10, currentPage: 5 });
+      expect(pagination).toStrictEqual([1, 2, "...", 4, 5, 6, "...", 9, 10]);
```

Comment:
This is ok in my opinion, but since we talked about it earlier, it would also make sense to render the 3 in place of the ellipsis.

Reply:
I really like to not render the ellipsis when it only hides one page. https://github.com/direktiv/direktiv-ui/pull/412/commits/1315ad16d481d7e7f03a7baa2b5ec6b099feaa99